cfgov/unprocessed/css/organisms/carousel.less

```diff
@@ -28,14 +63,37 @@
   &_item:nth-child(4) {
     background: forestgreen;
   }
+
+  &_thumbnails {
+    // Remove gap between child inline elements. See https://stackoverflow.com/questions/26950005/strange-gap-between-two-buttons
+    font-size: 0;
+  }
+
+  &_thumbnail {
+    width: 25%;
+    // Set font-size to override parent font-size:0, which removes the gap between the thumbnails.
+    font-size: unit( 16px / @base-font-size-px, rem );
+    background: @white;
+    border: 1px solid @gray;
+    border-right: none;
+
+    &:nth-child(4) {
+      border-right: 1px solid @gray;
+    }
+  }
+
+  &_thumbnail-selected {
+    background: @gray-10;
+  }
 }
 
 // Show no-js state of carousel where all items show and buttons are hidden.
 .no-js .o-carousel {
-  // Override u-hidden when 
+  // Override u-hidden when
   display: block !important;
 
-  &_btn {
+  &_btn,
+  &_thumbnails {
     display: hidden !important;
```

Comment:
Should this be `none`?

```suggestion
    display: none !important;
```

I get "invalid property value" in Chrome, and the buttons/thumbnails still show even with JS turned off:
![image](https://user-images.githubusercontent.com/654645/69351106-bf9ae700-0c48-11ea-8130-97e89ac717f2.png)

Reply:
It should, thanks! 😬